Formula/cp2k.rb

```diff
@@ -1,9 +1,8 @@
 class Cp2k < Formula
   desc "Quantum chemistry and solid state physics software package"
   homepage "https://www.cp2k.org/"
-  url "https://github.com/cp2k/cp2k/releases/download/v6.1.0/cp2k-6.1.tar.bz2"
-  sha256 "af803558e0a6b9e9d9ce8a3ab955ba32bacd179922455424e061c82c9fefa34b"
-  revision 1
+  url "https://github.com/cp2k/cp2k/archive/v7.1.0.tar.gz"
+  sha256 "e244f76d7e1e98da7e4e4b2e6cefb723fa1205cfae4f94739413be74952e8b4e"
```

Comment:
```suggestion
  url "https://github.com/cp2k/cp2k/releases/download/v7.1.0/cp2k-7.1.tar.bz2"
  sha256 "ccd711a09a426145440e666310dd01cc5772ab103493c4ae6a3470898cd0addb"
```

We could try to use the same url scheme as it was before, it could contain all the submodules required